install-tests/react-native/install.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -e
+
+react_native_version="${1:-nightly}"
+realm_version="${2:-latest}"
+
+echo "Using React Native version = $react_native_version"
+echo "Using Realm version = $realm_version"
+
+APP_DIR="$PWD/`dirname $0`/app"
```

Comment:
```suggestion
APP_DIR="$PWD/app"
```
This can be simpler.  The output of `dirname $0` will just be `.`

Reply:
That would assume you're running the script from a specific working directory, in which case running `./install-tests/react-native/install.sh` from the root would put an `/app` in the root of the project.

I suppose this is a personal preference: When writing a node.js script I try to use directories relative to `__dirname` instead of using the working directory. This is the simplest way I found doing the same from a bash script.